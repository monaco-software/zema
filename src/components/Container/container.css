.container {
  display: flex;
  flex-direction: column;
  height: 100%;
  max-width: 1000px;
  margin: 0 auto;
  padding: 48px 96px;
}
