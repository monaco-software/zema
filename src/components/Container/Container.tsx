import './container.css';
import React, { FC, HTMLAttributes } from 'react';
import b_ from 'b_';

const block = b_.lock('container');

export const Container: FC<HTMLAttributes<HTMLDivElement>> = ({ className, ...restProps }) => {
  const classNameString = className ? ` ${className}` : '';
  const containerClass = `${block()}${classNameString}`;

  return (
    <div
      className={containerClass}
      {...restProps}
    />
  );
};

