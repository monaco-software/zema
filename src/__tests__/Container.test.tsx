import React from 'react';
import renderer from 'react-test-renderer';
import { Container } from '@components/Container/Container';

it('<Container /> should render content', () => {
<<<<<<< HEAD
  const tree = renderer.create(
    <Container>
      some content
    </Container>
  ).toJSON();
=======
  const tree = renderer.create(<Container>some content</Container>).toJSON();
>>>>>>> dev

  expect(tree).toMatchSnapshot();
});
