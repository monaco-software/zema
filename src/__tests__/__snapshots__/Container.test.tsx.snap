// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> should render content 1`] = `
<div
  className="container"
>
  some content
</div>
`;
